src/component/note-form.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import autoBind from '../utils/utils';
+
+export default class NoteForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: '',
+      content: '',
+    };
+    autoBind.call(this, NoteForm);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.handleAddNote(this.state); // eslint-disable-line
```

Comment:
This might be a good spot to reset your state back to empty strings so that the input fields get cleared upon submit.